examples/data_pipeline/README.md

```diff
@@ -36,31 +36,7 @@ This pipeline example shows the application with the vocab text file from Huggin
 
 The command to run the pipeline:
 
-    python pipelines.py --pipeline text_vocab 
-
-
-## Legacy PyText
-
-This pipeline example shows the application with the existing `ScriptVocab` in pytext library. The `ScriptVocab` instance is built from a text file where a column of vocab tokens are read in sequence.
-
-* `torchtext.experimental.transforms.BasicEnglishNormalize` backed by `re2` regular expression library
-* `from pytext.torchscript.vocab.ScriptVocabulary`
-
-With the dependency of `pytext` library, the command to run the pipeline:
-
-    python pipelines.py --pipeline pytext
-
-
-## Experimental PyText
```

Comment:
why do we want to remove this pipeline?

Reply:
The github repo of pytext is not maintained anymore and is not in good state as of this writing. This would mean that we have code in torchtext that is breaking going forward. I wasn't so sure, if we still want to maintain these code snippets that may sporadically break?

cc: @hudeven

Reply:
Let's remove this in a separate PR, since we're not entirely clear on it and the other changes in this diff can go ahead, plus we might need it for comparison relatively soon.